__tests__/reporter.spec.js

```diff
@@ -140,7 +140,7 @@ describe('reporter', () => {
                 }
             };
             jest.spyOn(utils, 'getAttributes').mockImplementation(() => 'attributes');
-            jest.spyOn(utils, 'getTestCaseId').mockImplementation(() => 'testCaseId');
+            jest.spyOn(utils, 'getValueByKey').mockImplementation(() => 'testCaseId');
```

Comment:
Is it necessary to mock `getValueByKey` method? It looks like a pure method and its result depends only on the arguments.

Reply:
Yes, it is. We need this mock as our expectedTestItemDataObj expects the result of execution the function `getValueByKey` (now it's getCollectionVariablesByKey)